roles/lib_openshift/src/doc/label

```diff
@@ -0,0 +1,64 @@
+# flake8: noqa
+# pylint: skip-file
+
+DOCUMENTATION = '''
+---
+module: oc_label
+short_description: Create, modify, and idempotently manage openshift labels.
+description:
+  - Modify openshift labels programmatically.
+options:
+  state:
+    description:
+    - State represents whether to create, modify, delete, or list
```

Comment:
@tbielawa,

This is a good observation.  The 'present' state in declarative means create or update.  Think crud.
create|update = present
delete = absent
read = list
Basically, make-it-so and the 'so' being the state I'm passing in.  For the labeling module, make the labels present on the object.  Note that we did go outside the general pattern here and included an 'add'.  This just adds a label if needed.  That way you don't have to pass _ALL_ labels each time.  
If they already exist, exit.  
If they don't exist, create them.
If they aren't exact, update them.

I understand that the ansible terms for these can be less than ideal but its the same for the base modules.  Take a look at the user module's state:
absent|present - Whether the account should exist or not, taking action if the state is different from what is stated.

Hopefully this clears it up.  I'd be happy to discuss further.